src/js/providers/html5.js

```diff
@@ -194,14 +205,19 @@ function VideoProvider(_playerId, _playerConfig, mediaElement) {
             const error = video.error;
             const errorCode = (error && error.code) || -1;
 
-            if (errorCode === 3 && _this.currentTime !== -1 && OS.iOS) {
-                // Workaround iOS bug https://bugs.webkit.org/show_bug.cgi?id=195452
+            if (errorCode === 3 || errorCode === 4 && _this.retries < 3) {
```

Comment:
So we would trigger this warning if `errorCode === 3` OR `errorCode === 4 && _this.retries < 3`?

Reply:
Nice catch. Fixed!